client/src/views/Home/HomePage/HomePage.tsx

```diff
@@ -1,40 +1,43 @@
+import React, { useState } from 'react';
 import { Box, Grid, Typography } from '@mui/material';
 import { SwiperSlide } from 'swiper/react';
-import React, { useState } from 'react';
-import DiscountCard from '../../../components/card/DiscountCard/DiscountCard';
+import DiscountCard from 'components/card/DiscountCard/DiscountCard';
 import PersonCard from '../../../components/card/PersonCard/PersonCard';
 import Discount from '../../../components/layout/discount-area/Discount';
 import ImageSwiper from '../../../components/swiper/ImageSwiper';
 import Title from '../../../components/Title/Title';
-import subscriptionData from '../../../data/subscriptionData';
-import personData from '../../../data/personData';
 import FitProgram from '../HomeFitProgram/HomeFitProgram';
 import PrimaryButton from '../../../components/Button/PrimaryButton/PrimaryButton';
 import Banner from '../HomeBanner/HomeBanner';
 import Benefits from '../HomeBenefits/HomeBenefits';
-import benefitsData from '../../../data/benefitsData';
-import questionsData from '../../../data/questionsData';
 import Questions from '../HomeQuestions/HomeQuestions';
 import { getWindowWidth } from '../../../utils/helpers/window-helper';
+import Panel from '../../../types/panel';
+import {
+  subscriptionData,
+  reviewData,
+  benefitsData,
+  questionsData,
+} from '../../../data/constants';
 
-const HomePage = () => {
-  const [expandedPanel, setExpandedPanel] = useState({
+const HomePage: React.FC = () => {
+  const [expandedPanel, setExpandedPanel] = useState<Panel>({
     1: true,
```

Comment:
weird logic, rethink how could you make it more effective. what if there will be 100properties? are you going to list it all?